pages/orders.tsx

```diff
@@ -11,14 +11,19 @@ const Orders: NextPage = () => {
 
   useEffect(() => {
     appState?.domain && appState?.token && fetch(
-      "/api/orders",
-      { headers: [
-        [SALEOR_DOMAIN_HEADER, appState.domain],
-        ["authorization-bearer", appState.token!],
-      ] },
+      "/api/number-of-orders",
```

Comment:
I see you changed the name of api endpoint - my intention was to have `pages/api/<name>` following `pages/<name>` and containing all necessary config in response.

Here you had:
 * component in `/pages/orders`
 * api route with configuration at `/api/pages/orders` responding with `{"number-of-orders": <value>}`

assuming, that we would want to have another value fetched for that component (let say `warehouse-id`) following pattern that you're trying to introduce here, should that value have separate API endpoint?

Reply:
I agree. @2can what was the intention behind the `route` change here?

Reply:
(one more thing to think about here, in future, is how we want to design a authorization framework to access these api routes and configuration in particular, as currently we just check JWT validity).

Reply:
Well my reasoning behind it was that currently the `api/orders` doesn't return `orders` but just tries to get metadata value.

This repo is targetted as a starting point but the wording for the orders example seems in this case confusing.

If you'd like to stick to the convention you've proposed I think it would be better to have a metadata example as a separate page with a corresponding `api` endpoint.

Reply:
> If you'd like to stick to the convention you've proposed

I'm not attached to my proposal and open for discussion here (that was the reason for that comment).

> I think it would be better to have a metadata example as a separate page with a corresponding api endpoint.

We have `/api/configuration`, I was even initially thinking about using it (I recall we discuss that briefly offline with @zaiste ), but end up with dedicated endpoint (called, because I've not founded better name, `/api/orders`) because I've anticipated, that at some point we would like to have different permissions attached to different components/endpoints and allowing all staff users to fetch all configuration values would be huge security issue (as I wrote above, we should have a discussion about how to secure these endpoints in consistent way).

Reply:
@2can @magul could we rename it from `/api/orders` to `/api/configuration/orders`? to underline it's about `Order` configuration?

Reply:
Eventually let's move to GraphQL, but I'd like to discuss it when you're back from holiday @magul

Reply:
> could we rename it from /api/orders to /api/configuration/orders? to underline it's about Order configuration?

I'm fine with that

Reply:
@zaiste @magul done